mini-project-1/products.json

```diff
@@ -2,20 +2,203 @@
   {
     "productId": 1,
     "productName": "Starbucks Espresso",
-    "price": 99,
+    "price": [59, 99, 129],
     "description": "Dark roasted beans for your delightful morning...",
-    "category": ["Dark roast", "SML"], //SML stands for available sizes
+    "category": ["Dark roast", "SML", "Starbucks"],
     "productImage": "assets/product_images/cafe1.png",
     "discount": "nodiscount",
+    "rating": "5"
   },
   {
     "productId": 2,
     "productName": "Starbucks Holiday",
-    "price": 99,
-    "description": "Holiday stuff, good stuff...",
-    "category": ["Light roast", "SL"],
+    "price": [59, 129],
+    "description": "Enjoy our special blend of balanced and sweet flavours with herbal and maple notes made with the same high quality 100% Arabica beans.",
+    "category": ["Medium roast", "SL", "Starbucks"],
```

Comment:
Remove the "SL" and all the other "SML". They are not a category.
Moreover there is a more elegant way to do it if we need to do it later.
We don't need to do that but consider some key value pairs like:

product_price: {
"S": 59,
"M": 79,
"L": 99,
}